templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -6,9 +6,11 @@
 	<div class="menu">
 		<div class="header">{{.ctxData.locale.Tr "repo.pick_reaction"}}</div>
 		<div class="divider"></div>
-		{{range $value := AllowedReactions}}
-			<a class="item reaction" data-tooltip-content="{{$value}}" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
-		{{end}}
+		<div class="reactions-grid">
+			{{range $value := AllowedReactions}}
+				<a class="item reaction" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
```

Comment:
I think this breaks a11y and Fomantic UI standard.

The standard layout is `.menu > .item`

Reply:
Keyboard navigation via up/down arrows still works. It's kind of not intuitive thought because left/right do not work as one would expect. So I don't think this keyboard navigation is really of any use here.

Reply:
I think we can keep this extra div. It does not break at least the keyboard navigation, so I'm quite confident it should work as expected.

Reply:
Done, actually removed it and  restored old shitty css.